src/test/bare-module-worker_test.ts

```diff
@@ -1039,6 +1039,88 @@ suite('bare module worker', () => {
     await checkTransform(files, expected, {}, cdn);
   });
 
+  test('prefers "import" + "browser" condition exports over default', async () => {
+    const files: SampleFile[] = [
+      {
+        name: 'index.js',
+        content: `
+          import 'foo';
+          import 'foo/bar.js';
+        `,
+      },
+    ];
+
+    const cdn: CdnData = {
+      foo: {
+        versions: {
+          '1.0.0': {
+            files: {
+              'browser/index.js': {
+                content: 'BROWSER',
+              },
+              'browser/bar.js': {
+                content: 'BROWSER',
+              },
+              'default/index.js': {
+                content: 'DEFAULT',
+              },
+              'default/bar.js': {
+                content: 'DEFAULT',
+              },
+              'package.json': {
+                content: `{
+                  "exports": {
+                    ".": {
+                      "import": {
+                        "browser": "./browser/index.js"
+                      },
+                      "default": "./default/index.js"
+                    },
+                    "./bar.js": {
+                      "import": {
+                        "browser": "./browser/bar.js"
+                      },
+                      "default": "./default/bar.js"
```

Comment:
what about exports without `"import"`? ie, in Lit we use default and development, conceivably one of them could have a browser variant.

Reply:
There are tests above this one that cover single export conditions, like just "imports" and just "development". I think given the implementation (an unordered set of conditions), we don't need to test all permutations (i.e. just "development" should work exactly like just browser").